pkg/engine/source/filesystem.go

```diff
@@ -213,6 +218,7 @@ func ReadMetadata(queryDir string) map[string]interface{} {
 
 		return nil
 	}
+	defer f.Close()
```

Comment:
gosec created a [new issue](https://github.com/Checkmarx/kics/security/code-scanning/9165?query=ref%3Arefs%2Fpull%2F2410%2Fmerge) pointing to this line